conda/cli/main_remove.py

```diff
@@ -171,6 +171,21 @@ def execute(args: Namespace, parser: ArgumentParser) -> int:
         msg = "Cannot remove current environment. Deactivate and run conda remove again"
         raise CondaEnvironmentError(msg)
 
+    # Get the PrefixData for the default_activation_env; this can either be
+    # a prefix or an env name, so we try both here.
+    default_activation_env_pfd = PrefixData(context.default_activation_env)
+    if not (
+        default_activation_env_pfd.exists()
+        and default_activation_env_pfd.is_environment()
+    ):
+        default_activation_env_pfd = PrefixData.from_name(
+            context.default_activation_env
+        )
```

Comment:
let's add a new context property (e.g., `context.default_activation_prefix`) that does the work of testing for an absolute path versus calling `locate_prefix_by_name`